### ISSUE TITLE
Corrected image is blurry

### ISSUE THREAD

**Issue by xingzhe1129** [czimaginginstitute/AreTomo3#16]
![Screenshot 2024-11-04 at 18 10 33](https://github.com/user-attachments/assets/8f186ef5-3ea0-4449-ba56-7a228a8ea159)
Hi, I ran Aretomo3 with recommended parameters but the corrected is much more blurry. Does anyone know what happened. Thanks.

**Comment by szhengczii** [comment]
I need more info to diagnose this problem including command line parameters. Is this image after motion correction?

**Comment by xingzhe1129** [comment]
Yes. This image is after motion correction. My command line parameters are AreTomo3 -InMdoc Position_8.mdoc -Gain gain.mrc -OutDir Aretomo3 -McPatch 5 5 -McBin 1 -PixSize 2 -AtBin 4 -AtPatch 5 5 -FlipVol 1 -OutImod 1 -Wbp 1 -Gpu 0 1 2 3 4 5 6 7 -CorrCTF 1. It seems that the corrected one is just stacked together but there is only one frame in the corrected one. I also did motion correction with patch motion correction in cryosparc for this tomo data and the corrected images look great.

**Comment by szhengczii** [comment]
@xingzhe1129 Please try it again with "-Group 2 4" to see if it helps.
